src/Groups/action.jl

```diff
@@ -291,9 +291,13 @@ end
     on_indeterminates(f::MPolyIdeal, p::MatrixGroupElem)
 
 Return the image of `f` under `p`.
-If `p` is a `PermGroupElem` then it acts via permuting the indeterminates,
-if `p` is a `MatrixGroupElem` then it acts via evaluating `f` at the
+
+If `p` is a `PermGroupElem` then it acts via permuting the indeterminates.
```

Comment:
Should we split this doc string for `on_indeterminates` between `PermGroupElem` and `MatrixGroupElem`?